src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -543,7 +561,20 @@ export default function LoadedRow({
         volume={<ClickableContent>{volume ? formatDollarAmount(volume ?? undefined) : '-'}</ClickableContent>}
         sparkLine={
           <SparkLine>
-            <ParentSize>{({ width, height }) => <SparklineChart width={width} height={height} />}</ParentSize>
+            <ParentSize>
+              {({ width, height }) => (
+                <LineChart
+                  data={pricePoints}
+                  getX={(p: PricePoint) => widthScale(p.timestamp)}
+                  getY={(p: PricePoint) => rdScale(p.value)}
+                  curve={curveCardinal.tension(curveTension)}
+                  color={pricePercentChange && pricePercentChange < 0 ? theme.accentFailure : theme.accentSuccess}
+                  strokeWidth={2}
+                  width={width}
+                  height={height}
+                />
+              )}
+            </ParentSize>
           </SparkLine>
```

Comment:
Can we wrap all of this in a separate imported component instead, including all the vars from L487-497

Reply:
ooh yes moved everything to sparkline charts file